src/girouette/tw/border.cljc

```diff
@@ -1,10 +1,204 @@
 (ns girouette.tw.border
-  (:require [girouette.tw.common :refer [value-unit->css]]))
+  (:require [garden.selectors :as gs]
+            [girouette.tw.common :refer [value-unit->css value->css read-number dot default-pipeline]]
+            [girouette.tw.color :refer [read-color color->css]]))
 
 (def components
-  [{:id :xxx
-    :rules "
-    xxx = <'xxx'> integer
+  [{:id     :border-radius
+    :rules  "
+    border-radius = <'rounded'>  (<'-'> border-radius-position)? (<'-'> border-radius-size)?
+    border-radius-position  = border-radius-side | border-radius-corner
+    border-radius-side = 't' | 'r' | 'b' | 'l'
+    border-radius-corner = 'tl' | 'tr' | 'br' | 'bl'
```

Comment:
The cases for `border-radius-side` and `border-radius-corner` can be handled together, I suggest to regroup them into the `border-radius-position` rule. It will also make the handling function simpler.